### ISSUE TITLE
Improve CKEditorWebpackPlugin to handle context and plurals in translations.

### ISSUE THREAD

**Issue by phaux** [ckeditor/ckeditor5#6204]
## 📝 Provide a description of the improvement

Improve CKEditorWebpackPlugin to handle context and plurals in translations. Right now only regular messages are supported.

Related: https://github.com/ckeditor/ckeditor5/issues/6158

**Comment by phaux** [comment]
## Overview of the changes

### Converting PO to JSON

`createDictionaryFromPoFileContent` is a function which parses a PO file and outputs JSON. The original implementation creates a simple mapping from `msgid` to `msgstr`. The new functionality will support extracting `msgctxt` and multiple plural forms as well. It's also important to include the content of the "plural-forms" header in the output.

Example extracted JSON:

```JSON
{
  "PLURAL_FORMS": "nplurals=3; plural=(n == 1) ? 0 : (n < 5) ? 1 : 2",
  "message id": ["localised message", "localised message plural 1", "localised message plural 2"],
  "message context|message id": "localised message"
}
```

### Multiple language translation service

MultipleLanguageTranslationService class will be simplified significantly. It won't replace the message IDs with short IDs anymore. It doesn't even need to look at the source code. The only thing it should do is to include the assets generated from the PO files.

The reasoning behind that was that the full message IDs should compress fairly well when using gzip anyways because they contain a lot of words that are used in the code around them. The other reason was that since we would need to handle the message context as well, the generated short IDs would actually have even more entropy than the original two strings. (e.g.: `ct( 'command: add table column', 'table column' )` vs `ct( 'zY5x', '2Fg' )`) and might even make the compression worse.

**Comment by ma2ciek** [comment]
The issue was closed by ckeditor/ckeditor5-dev#614.
